hal/network/ncp_client/sara/sara_ncp_client.h

```diff
@@ -135,6 +136,7 @@ class SaraNcpClient: public CellularNcpClient {
     void parserError(int error);
     void resetRegistrationState();
     void checkRegistrationState();
+    int interveneRegistration();
```

Comment:
I'll leave this as-is, as I feel it states the intent more clearly:
> take part in something so as to prevent or alter a result or course of events.

Whereas recovery would sound more like we've already lost registration and attempt to recover, which is not exactly what's happening (at least not in all the cases).